### PR TITLE
[Revised] Adding MarkBind as an alternative docs option

### PR DESCRIPTION
**Pull request by damithc**
A revised version based on #156 
Once finalized, all commits will be squashed to one commit and the commit message will mention #156 as the primary source of changes
Deployed site https://damithc.github.io/ab3-markbind/index.html

### CODE REVIEW

**Comment by damithc** [comment]
@tlylt can have a look?

File: .github/workflows/docs.yml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+        with:
+          java-version: '11'
+          distribution: 'temurin'
+      - name: Build & Deploy MarkBind site
+        uses: MarkBind/markbind-action@v2
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+          rootDirectory: './docs'
+          baseUrl: '/addressbook-level3' # replace with your repo name
+          version: '^5.0.2'
```
Comment (tlylt) [inline .github/workflows/docs.yml]:
```suggestion
          version: '^5.1.0'
```

**Comment by tlylt** [comment]
> @tlylt can have a look?

Other than the above, LGTM!

**Comment by damithc** [comment]
Released to students as an option (see step 4 [here](https://nus-cs2103-ay2324s1.github.io/website/admin/appendixE-gitHub.html#:~:text=the%20developers%20team.-,tP%20Team%20Repo%20Setup,-(/)%20Only))
Closing this PR as it is never meant to be merged. We can observe how things go this semester and revise further in the following semester.

Thanks again @tlylt for your immense help with this.  💯

**Pull request closed without merging**